README.md

```diff
@@ -130,8 +134,8 @@ entities (no need to draw an ERD):
 * Pet
 
 Your answer:
-```
-Replace this with your answer
+```text
+ERD stands for Entity Relationship Diagram.  An ERD is a tool that's used to visualize how data relate to one another within the structure of a database program.  One genie might have lived in many lamps, and one person might have possession of many lamps.  Also, one lamp may have many (past) owners, and one genie can have granted wishes to many people.
```

Comment:
:+1:
Looks good.  I was expecting some attributes for the Entities, in addition to your provided relationships.